co-authors-plus.php

```diff
@@ -1052,22 +1052,61 @@ function filter_count_user_posts( $count, $user_id ) {
 
 		if ( is_object( $user ) ) {
 			$user = $this->get_coauthor_by( 'user_nicename', $user->user_nicename );
-
 			$term = $this->get_author_term( $user );
 
-			if ( $term && ! is_wp_error( $term ) ) {
-				if ( 'guest-author' === $user->type ) {
-					// If using guest author term count, add on linked user count.
-					$count = (int) $count + $term->count;
-				} else {
-					$count = $term->count;
-				}
+			if ( strlen( $user->linked_account ) > 1 ) {
+				return $this->get_linked_post_count( $user->user_nicename, $user->linked_account );
+			}
+
+			if ( isset( $term->count ) && null !== $term->count ) {
```

Comment:
Are you seeing any cases where `$term->count` is a null value?

Reply:
See https://github.com/Automattic/Co-Authors-Plus/pull/776#discussion_r613196898.